#! /bin/sh /usr/share/dpatch/dpatch-run
## wvdvi-output-ext.dpatch by Matej Vela <vela@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: wvDVI: Avoid `are the same file' error when the output extension
## DP: is `.dvi' (#309524).

@DPATCH@
diff -urNad wv-1.2.1~/wvDVI.in wv-1.2.1/wvDVI.in
--- wv-1.2.1~/wvDVI.in	2006-03-18 16:13:32.000000000 +0100
+++ wv-1.2.1/wvDVI.in	2006-03-18 16:16:15.000000000 +0100
@@ -299,5 +299,4 @@
 	exit 1
 fi
 
-cp -f "$name".dvi "$o_file"
-rm -f "$name".dvi
+[ "$name".dvi -ef "$o_file" ] || mv -f "$name".dvi "$o_file"
